api/src/test/resources/org/openmrs/api/include/ObsServiceTest-initial.xml

```diff
@@ -11,6 +11,14 @@
 
 -->
 <dataset>
+
+  <person person_id="9" gender="" birthdate="1948-01-01 00:00:00.0" birthdate_estimated="0" dead="0" creator="1" date_created="2005-01-01 00:00:00.0" voided="0" uuid="bed9239a-8dfd-11e0-9188-705ab6a580e0"/>
+  <person_name person_id="9" person_name_id="101" preferred="1" creator="1" date_created="2005-01-01 00:00:00.0" voided="0" given_name="qwerty11" middle_name="O" family_name="One"  uuid="d8744a76-90de-11e0-9c16-705ab6a580e0"/>
```

Comment:
A person already exists with id = 9 in [the standard test data set](https://github.com/openmrs/openmrs-core/blob/3b570b8c9290e60c505d51d6272884d533f5a1ba/api/src/test/resources/org/openmrs/include/standardTestDataset.xml#L243). Do we need to add all this here?

Reply:
@ibacher   thanks ian,i get to see that we use a different dataset than the standard test data set ie ObsServiceTest-initial.xml  
and am of a view that all tests are from that data set, could that be true?

Reply:
So, the `BaseContextSensitiveTest` class [automatically loads](https://github.com/openmrs/openmrs-core/blob/3b570b8c9290e60c505d51d6272884d533f5a1ba/api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java#L900) both the `initialInMemoryTestDataSet.xml` and the `standardTestDataset.xml` files as long as you haven't annotated the test class with the `@SkipBaseSetup` annotation. So, in other words, you should have the data in those two files available to you as well when running these tests.

Reply:
pont noted,thanks @ian,how do my new changes on this look to you